fhir_river_api/src/app.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+import os
+from flask import Flask
+from flask import request
+from flask_restful import Api
+
+from fhir_river_api.src.config.logger import create_logger
+from fhir_river_api.src.errors import OperationOutcome
+from fhir_river_api.src.producer_class import RiverApiProducer
+import time
+
+logger = create_logger("fhir_river_api")
+
+# Create flask app object
+app = Flask(__name__)
+api = Api(app)
+
+producer = RiverApiProducer(broker=os.getenv("KAFKA_BOOTSTRAP_SERVERS"))
+
+
+@app.route("/run_sample", methods=["POST"])
+def trigger_sample_extractor():
+    """
+    Extract all records for the specified resource
+    """
+    body = request.get_json()
+    resource_id = body.get("resourceId", None)
+    primary_key_values = body.get("primaryKeyValues", None)
+    batch_id = "batch_{time_sec}".format(time_sec=round(time.time()))
+
+    try:
+        create_extractor_trigger(resource_id, batch_id, primary_key_values)
+        return "Success", 200
+
+    except Exception as e:
+        raise OperationOutcome(e)
+
+
+@app.route("/run_batch", methods=["POST"])
+def trigger_batch_extractor():
+    """
+    Extract all records for the specified resource
+    """
+    body = request.get_json()
+    resource_ids = body.get("resourceIds", None)
+    batch_id = "batch_{time_sec}".format(time_sec=round(time.time()))
```

Comment:
If the batch id is rounded to the closest second, aren't you afraid that some batch could have the same id in the loop just below?

Reply:
Can we use UUID here, the same way you used it for Pyrog @Jasopaum ?

Reply:
I don't have any strong opinion on that, I guess UUID is ok

Reply:
Otherwise int(time.time() * 1000)

Reply:
Yep this is sth I'd like to discuss. We can concatenate resource_ids + seconds (or milliseconds)
I'm fine with using your solution. How do you make sure it's always unique when you have to restart your service?

Reply:
https://stackoverflow.com/questions/1155008/how-unique-is-uuid

Reply:
I guess it is pretty safe :)